src/pages/privacy/index.tsx

```diff
@@ -583,7 +584,7 @@ export const PrivacyPage = () => {
           THIS PRIVACY POLICY IS SUBJECT IN ALL RESPECTS TO THE TERMS OF USE
           AVAILABLE AT{' '}
           <a
-            href="https://app.carbondefi.xyz/terms"
+            href={config.appUrl + '/terms'}
```

Comment:
Dont we have these links in the externalLinks object?

Reply:
Should also change the text HTTPS://APP.CARBONDEFI.XYZ/TERMS underneath.